src/main/java/com/gmail/inverseconduit/javadoc/ClassInfoXmlParser.java

```diff
@@ -0,0 +1,179 @@
+package com.gmail.inverseconduit.javadoc;
+
+import java.util.Arrays;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import com.gmail.inverseconduit.utils.DocumentWrapper;
+
+/**
+ * Parses {@link ClassInfo} objects from XML documents.
+ * @author Michael Angstadt
+ */
+public class ClassInfoXmlParser {
```

Comment:
I really like adding Javadocs to things, even if they are superfluous.  That's just my style.  It forces me to explain what the class/method does and why it's there.  I feel more confident working with code that is documented well.